builtin/logical/transit/path_encrypt.go

```diff
@@ -568,6 +573,23 @@ func (b *backend) pathEncryptWrite(ctx context.Context, req *logical.Request, d
 	return batchRequestResponse(d, resp, req, successesInBatch, userErrorInBatch, internalErrorInBatch)
 }
 
+func nonceAllowed(p *keysutil.Policy) bool {
```

Comment:
Do we need to consider managed keys here?

Reply:
Hmm, with managed keys we don't know if a nonce is needed or not, right?

Reply:
I believe that is correct, but with this it will never make it through.

Reply:
correct. especially in PKCS#11, it depends on the mechanism used.